src/Redux/configureStore.js

```diff
@@ -0,0 +1,13 @@
+import { combineReducers } from 'redux';
+import { configureStore } from '@reduxjs/toolkit';
+/* eslint-disable import/extensions */
```

Comment:
- [ ] Kindly remove this line  & solve the linter manually. As this is going to be mandatory in the next milestone.